raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -1220,6 +1225,27 @@ def handle_onchain_secretreveal(
     return iteration
 
 
+def handle_onchain_batch_unlock(
+        mediator_state,
+        onchain_batch_unlock,
+        channelidentifiers_to_channels,
+):
```

Comment:
This is incorrect IMO, a transfer pair has a *pair* of channels involved in it, if one of the channels is closed, it doesn't mean the other doesn't have anything else to process.

For instance, if we have a A-B-C path, and the channel B-C is closed by node A is offline. If the transfer pair is removed then once A goes back online it will try to send a remove expired lock which will never be processed and the channel will stuck. In the other direction the problem is different, then I believe the remove expire lock will not be sent and the channel will degrade.